site/blog/2022/clean-your-aws-account.mdx

```diff
@@ -28,12 +28,13 @@ Please note that our engine can offer this cleanup only for the resources covere
 ```sql TheButton[AWS account cleanup]="Clean your AWS Account"
 -- Installing all IaSQL modules
 SELECT iasql_install(variadic array(select module_name from iasql_modules_list()));
+SELECT iasql_begin();
 -- Deleting all resources from the db
 SELECT * FROM delete_all_records();
--- Previewing changes in your account
-SELECT * FROM iasql_preview_apply();
--- Applying delete changes, please uncomment it to perform the cleanup
---SELECT * FROM iasql_apply();
+-- Rollback the changes
+SELECT iasql_rollback();
```

Comment:
why would we want to roll back the changes? I think you meant to have this be `preview`?

Reply:
Oh I thought `preview` function was deleted. You're right.

Reply:
we should remove or at least comment `rollback` otherwise this snippet won't actually do anything